SilentUpdate/src/main/java/org/netbeans/modules/autoupdate/silentupdate/Activator.java

```diff
@@ -23,12 +24,26 @@ public void restored()
         exector.scheduleAtFixedRate(doCheck, 10000, 10000, TimeUnit.MILLISECONDS);
     }
 
-    private static final Runnable doCheck = () ->
+    private static final Runnable doCheck = ()
+            ->
             {
                 if (UpdateHandler.timeToCheck())
                 {
                     UpdateHandler.checkAndHandleUpdates();
+                    uninstallRemovedModules();
```

Comment:
i think that this should rather be part of the checkAndHandleUpdates flow. it would also spare a couple of module lookups. since we use a silentupdater, i dont think we ever want any control over when to call install, update, disable, uninstall etc. it can be part of a later commit however.